packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/sdk/test/arm-test/review/arm-networkanalytics.api.md

```diff
@@ -426,8 +426,8 @@ export interface NetworkAnalyticsClientOptionalParams extends ClientOptions {
 
 // @public
 export interface Operation {
-    actionType?: ActionType;
-    readonly display?: OperationDisplay;
+    readonly actionType?: ActionType;
```

Comment:
how about this one?

Reply:
https://github.com/Azure/typespec-azure/pull/1881/files#diff-823f10c1812ffa803f58da35c7d41623a79c2e5c37f0c15d09ce66bcfdcfbe97